src/sentry/sentry_metrics/aggregation_option_registry.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+from typing import Optional
+
+
+class AggregationOption(Enum):
+    HIST = "hist"
+
+
+METRIC_ID_AGG_OPTION = {
+    "d:transactions/measurements.fcp@millisecond": AggregationOption.HIST,
```

Comment:
This is a design choice we made in the beginning, where we want to be really opinionated with what options are available so people _cannot_ mix and match.

Reply:
> we don't know what aggregation options we want to make available in the future and how they will interact with each other

Wouldn't the aggregation options be mutually exclusive with each other? Can you provide example scenarios where this can cause problems?

Reply:
Aggregation options that effects retention days and what time rollup to perform would come in mind.

If we have aggregation options like `super_volatile_alert` where only 1 minute roll up is performed with TTL for 7 days. And another aggregation option like `monthly_trends` where only 1 day roll up is performed with some other TTL.

It wouldn't make sense to allow them both together.